flask_admin/model/base.py

```diff
@@ -10,7 +10,7 @@
 from jinja2 import contextfunction
 from wtforms.fields import HiddenField
 from wtforms.fields.core import UnboundField
-from wtforms.validators import ValidationError, Required
+from wtforms.validators import ValidationError, InputRequired
```

Comment:
http://wtforms.readthedocs.org/en/latest/whats_new.html says: "wtforms.validators.Required is now wtforms.validators.DataRequired"

Shouldn't this be DataRequired instead?